docs/diagrams/ModelClassDiagram.puml

```diff
@@ -36,16 +45,25 @@ AddressBook .up.|> ReadOnlyAddressBook
 
 ModelManager .up.|> Model
 Model .right.> ObservableList
+Model .right.> ObservableMap
+ObservableList -[hidden]down- ObservableMap
 ModelManager o--> "1" AddressBook
 ModelManager o-left-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
+AddressBook *.left.--> "1" GroupHashMap
+
+GroupHashMap o--> "*" Group
+
 UniquePersonList o--> "*" Person
 Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+Person *--> Birthday
+Person --> Event
```

Comment:
Would be good to write docs on the specifics for `Event` and `Picture` as it is not clearly shown in the class diagram. Like the multiplicity of Person to Event (1 to *) and multiplicity of Person to Picture (1 to 0..1)